tests/test_modeling_common.py

```diff
@@ -138,6 +138,34 @@ def _config_zero_init(config):
 TINY_BERT_FOR_TOKEN_CLASSIFICATION = "hf-internal-testing/tiny-bert-for-token-classification"
 
 
+def _run_torch_jit(in_queue, out_queue):
```

Comment:
What I don't "like" here is that now a test would fail for torchscripting before failing for output mismatch.

Reply:
I agree. I could even running the match in this function, return the matching results to the parent process, and fail the tests there - if you prefer.

Reply:
> I think it's safe to only run those tests on CPU. Also, when running locally it takes ~ 1 min (althought I agree my machine might be more powerful).

Do you run against my branch?

Reply:
Yes it would be perfect!
I did not run anything on your branch.

Reply:
It is indeed fast. This PR is not addressing the time issue, but the memory issue. Currently, each call to `test_torch_fx` will increase the memory usage by ~15MB.

The time issue comes after the fix, as we create new processes to run (part of) the code. Using `fork` is fine, but `spawn` will be quite slow. But currently, this time issue is insignificant (and `spawn` is only used on GPU CI running on our own runners - therefore no a real constraint).